VisionProSurgery/Resources/Localizable.xcstrings

```diff
@@ -0,0 +1,147 @@
+{
+  "sourceLanguage" : "en",
+  "strings" : {
+    "" : {
+
+    },
+    "ACCESSIBILITY_ICON_LABEL" : {
+      "extractionState" : "manual",
```

Comment:
We might want to enable automatic extraction for all strings in this localization catalogue.